core/src/main/antlr4/com/databricks/labs/remorph/parsers/snowflake/SnowflakeParser.g4

```diff
@@ -144,6 +144,10 @@ otherCommand
     | unset
     | call
     | beginTxn
+    | procStatement
```

Comment:
```suggestion
    | declareCommand
```
otherwise we have a recursion of `call` and `executeImmediate` on the `otherCommand`.